app/src/main/java/com/example/dine_and_donate/Activities/HomeActivity.java

```diff
@@ -35,21 +50,33 @@
     private Fragment mProfileFragment = new ProfileFragment();
     public User mCurrentUser;
 
+    private Fragment mDefaultFragment;
+    public LatLng markerLatLng;
```

Comment:
can you group the public and private fields together like

private var1
private var2

public var3
public var4

so it's a little easier to read?